rpg.py

```diff
@@ -1,12 +1,33 @@
+import render
+import util
+
 class Player:
     max_hp = 100
-    def __init__(self, name, gender, max_hp, health, fatigue, defence, strength, speed, intelligence):
+    def __init__(self, name, gender, xp, max_hp, health, fatigue, defence, strength, speed, intelligence):
         self.name = name
         self.gender = gender
+        self.level = util.calculate_current_level(xp)
+        self.xp = xp
```

Comment:
Maybe consider swapping these around, and using `self.xp` for clarity:
```suggestion
        self.xp = xp
        self.level = util.calculate_current_level(xp)
```